src/pages/experiments/[experimentId]/data-processing/index.jsx

```diff
@@ -381,12 +381,13 @@ const DataProcessingPage = ({ experimentId, experimentData, route }) => {
         <Col style={{ flex: 1, display: 'flex', justifyContent: 'flex-start', alignItems: 'center' }}>
           <Row style={{ display: 'flex' }} gutter={16}>
             <Col>
+              {/* Should be just wide enough that no ellipsis appears */}
               <Select
                 value={stepIdx}
                 onChange={(idx) => {
                   changeStepId(idx);
                 }}
-                style={{ fontWeight: 'bold', width: 270 }}
+                style={{ fontWeight: 'bold', width: 290 }}
```

Comment:
is there a way to make the width fit with the current length of the text without having to hardcode a particular width?

Reply:
It's annoying but not that I could find - I mentioned this in #346.

Reply:
ok, fair enough